internal/server/option/auth.go

```diff
@@ -404,6 +404,7 @@ func (a *authInterceptor) authenticate(
 				Claims:                     c,
 				ServiceAccountsByNamespace: sa,
 				BearerToken:                rawToken,
+				Username:                   a.cfg.OIDCConfig.UsernameClaim,
```

Comment:
Instead of putting the name of the claim here, let's put the _value_ of the claim.

Reply:
```suggestion
				Username:                   c[a.cfg.UsernameClaim],
```

If this somehow ends up being the empty string (like if the claim the operator wanted to use for this purpose didn't actually exist), it's fine, because we know the function that formats actor information will fall back on its old strategy of directly going after `email` and `sub` claims.